src/plugins/auto/src/schedule.cpp

```diff
@@ -85,8 +85,13 @@ void Schedule::generate_workers(const std::string& device, const SoCompiledModel
         OPENVINO_THROW("Every device used with AUTO should support query optimal_number_of_infer_requests property from compiled model ",
                     iie.what());
     }
-    const auto num_requests = (m_context->m_device_priorities.end() == it_numrequests ||
-                              it_numrequests->num_requests_per_devices == -1) ? optimal_num : it_numrequests->num_requests_per_devices;
+    auto num_requests =
+        (m_context->m_device_priorities.end() == it_numrequests || it_numrequests->num_requests_per_devices == -1)
+            ? optimal_num
+            : it_numrequests->num_requests_per_devices;
+    num_requests = num_requests <= 1 && m_context->m_performance_hint == ov::hint::PerformanceMode::THROUGHPUT
```

Comment:
why tput here?

Reply:
As we discussed offline earlier, not having this option will cause a hang.

Reply:
no, that was about cumulative tput, is not it? otherwise, how can you expect this PR to fix the customer issue which is in latency mode

Reply:
sorry, I got it. will debug why it caused a hang without this option.